File: vimeo-networking/src/main/java/com/vimeo/networking/model/Email.java
package com.vimeo.networking.model;

import com.google.gson.annotations.SerializedName;
import com.vimeo.stag.UseStag;

import java.io.Serializable;

/**
 * An email that the user has verified.
 * <p/>
 * Created by anthonyrestaino on 6/16/16.
 */
@SuppressWarnings("unused")
@UseStag
public class Email implements Serializable {

    private static final long serialVersionUID = -4112910222188194649L;

    @SerializedName("email")
    protected String mEmail;

    public String getEmail() {
        return mEmail;
    }
}

File: models/src/main/java/com/vimeo/networking2/Email.kt
package com.vimeo.networking2

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import java.io.Serializable

/**
 * User's email.
 */
@JsonClass(generateAdapter = true)
data class Email(

        @Json(name = "email")
        val email: String? = null

) : Serializable {

    companion object {
        private const val serialVersionUID = -1440735877L
    }
}
